### ISSUE TITLE
New Part Series: LanderCore Horizontal landers

### ISSUE THREAD

**Issue by shadowmage45** [shadowmage45/SSTULabs#466]
This part series will be a bit more along the traditional KSP theme than most SSTU parts, consisting of a set of parts with a common form-factor.   

Part-count reduction will still be a goal, but will be accomplished through integrating functions into the parts wherever possible (rather than concentrating on single-part craft); craft should generally end up in the 3-7 part range for fully functional landers, including science/transmitters/cargo bays (add 1-3 parts for multi-stage).

Overall the design of these parts will be very similar to the legacy SC-C parts, which can be seen here:
http://imgur.com/a/vibAM
(-very- similar, only with updated geometry and texturing procedures)

### Goal of Parts
* Enable consistently styled horizontal lander designs.
* Capable of standard crew-landers, cargo-landers, or module/rover delivery.
* Optional use of parts for orbital freighters, simply omit landing legs and landing engines
* Dockable/replaceable drop-tanks for semi-reusable landers.  Simply replace the drop tank(s), and re-use the majority of the hardware.
* Aeroshells / horizontal lander heat-shields to allow use on atmospheric bodies
  * How to allow for engine-thrust capability while shroud is attached to vessel? (visual/geometry/mesh)

### Preliminary part list workup
* Cockpit - Nose
* Cockpit - Inline
* Cargo Ramp - Nose/Tail
* Drop Tank Truss(es)
* Fuel Segment(s)
* Landing Leg / Landing Engine block
* Orbital/Service Engine block

### Preliminary design/aesthetic concepts
* Space 1999 Eagle
* SC-IC
* Nasa MSEV
* Babylon-5 Shuttlecraft

![](http://imgur.com/Yqaupgv.png)
![](https://s-media-cache-ak0.pinimg.com/originals/33/8b/4f/338b4ff1c5e6a61ae8551a312cddc1d2.jpg)
![](https://s-media-cache-ak0.pinimg.com/originals/86/59/2b/86592b9bcaf8ec25955a773dd4d89e86.jpg)
![](http://www.daemonstorm.com/images/completed_shuttle.png)

**Comment by taterkerman** [comment]
I haven't tried roverdude's horizontal craft, but how well do they deal with balance while landing?

**Comment by shadowmage45** [comment]
Roverdudes FTT 'HoneyBadger' series is the only one I have experience with, and not in recent game versions.

Using it with stock flight controls and it is very usable as long as the craft is designed close to perfectly balanced.  Small variations can be dealt with through engine gimbal, but large variations require manual thrust balancing between engines (which gets worse with asymmetric fuel distribution).

Essentially they 'don't deal with balance during landing', and leave that entirely up to the user as design and/or piloting choices.

Supposedly Throttle Controlled Avionics works very well for those type of craft setup, but I've never had time to play around with that mod.

**Comment by Jimbodiah** [comment]
TCA will work nicely landing VTOL craft. Last time I used it was in 1.0.5 though, but it worked fine.

**Comment by taterkerman** [comment]
TCA totally works, so cool!

![tca test](https://user-images.githubusercontent.com/17967275/27060123-e62ddc3a-4f97-11e7-9ccd-9b020a3d80bc.png)

**Comment by Jimbodiah** [comment]
![sstu_lander1](https://user-images.githubusercontent.com/16107012/27089467-b053a0c2-505a-11e7-9da1-3ea961024827.jpg)

I smell the new lander core

**Comment by taterkerman** [comment]
Rotate the CM so the windows look down...

**Comment by taterkerman** [comment]
Such airless world lander parts also can work well for assembly tugs, etc. at stations. 

Or stuff like ACES/XEUS.

**Comment by shadowmage45** [comment]
@SixDasher Yep, that is pretty much the main concept behind the 'horizontal lander' parts.  Only I'm not sure if I want to use standard 'round' stuff, or go with a custom form-factor (wide octogonal).  Perhaps both?  The octagonal would be a bunch of new parts, but the 'round' could use a bunch of the existing stuff, only needing new parts for the cargo bays and lander-engines.

@taterkerman Indeed; they would not be solely lander parts.  If you omit the HTOL landing engines/legs, they would work very well for orbital tugs or even interplanetary freighters.  Think along the lines of RoverDude's FTT mod, but being much more reasonable with the part-masses.


Any thoughts on what sizes of craft would be desirable?  I'm thinking 2.5m at the very minimum, though Ideally I want to be able to deliver 3.75m or 5m base modules/freight containers.  Perhaps 2.5m and 3.75m 'round' versions (for aero use), with a 5m+ octagonal version (vacuum use only).

**Comment by Jimbodiah** [comment]
The 2.5m-5m range would indeed be optimal. 

btw: I just made a lander with what was available, didn't mean anything by it like that it needed to be round.

A 2.5m round version would be ideal for the COS/DOS parts. And the round version would allow you to make the Constellation lander.

In all honesty though, this would have made more sense as a station builder vs the shuttle. Just send up a conventional rocket with an orbital section that was expendable, minus the crew return capsule. The SpaceX route of a reusable first stage would have made more sense.

**Comment by taterkerman** [comment]
Whatever the form factor, I like the skycrane capability---and I mean that in the Sikorsky Skycrane sense.

I tend to see those octagonal parts as farther future (in _look_, anyway) than the other parts. Having the crew parts be multitasks in terms of use seems like it would save you some other work, as well. Blunter nose, with windows---like the modern pressurized rover designs that require excellent view of the terrain in front/below the vehicle.
 
Such crew parts have many possible uses. You are entirely right about the ability to land base parts. I tried to use the saddle truss on mine, but the nodes on that confuse me, lol.

I'll poke around for pictures.

![nasanas1616117605224325](https://user-images.githubusercontent.com/17967275/27114442-f416dfe2-5080-11e7-9ed6-3070a0712173.jpg)

![00-phoenix](https://user-images.githubusercontent.com/17967275/27114443-f87a6c2a-5080-11e7-92c2-3d133f6f63c8.jpg)

**Comment by taterkerman** [comment]
https://ntrs.nasa.gov/archive/nasa/casi.ntrs.nasa.gov/20160014035.pdf

**Comment by shadowmage45** [comment]
That's a new one, and an interesting concept/layout.  Seems very Kerbal in its staging setup (full MAV inside cargo bay).  Might make a good concept for a new ShipCore/LanderCore part series.

**Comment by taterkerman** [comment]
Yeah, when I see anything that might be useful, I always want to drop it someplace where it can be found again :D 

The ascent vehicle is a lot like the core of Altair (the vertical cylinder variant).
